src/wallet/wallet.cpp

```diff
@@ -2667,14 +2694,14 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
     }
 
     // List of values less than target
-    std::pair<CAmount, std::pair<const CWalletTx*,unsigned int> > coinLowestLarger;
-    coinLowestLarger.first = fUseInstantSend
-                                        ? sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE)*COIN
-                                        : std::numeric_limits<CAmount>::max();
-    coinLowestLarger.second.first = NULL;
-    std::vector<std::pair<CAmount, std::pair<const CWalletTx*,unsigned int> > > vValue;
+    boost::optional<CInputCoin> coinLowestLarger;
+    std::vector<CInputCoin> vValue;
     CAmount nTotalLower = 0;
 
+    if (fUseInstantSend) {
+        coinLowestLarger->txout.nValue = sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE) * COIN;
```

Comment:
This crashes. Not sure what's the best way to incorporate spork5 into new code tbh but since it's not going to be used in llmq IS checks anyway I would probably just go with
```
    // TODO: drop SPORK_5_INSTANTSEND_MAX_VALUE spork
    if (fUseInstantSend && nTargetValue > sporkManager.GetSporkValue(SPORK_5_INSTANTSEND_MAX_VALUE) * COIN) {
        return false;
    }
```